calico-enterprise/getting-started/upgrading/upgrading-enterprise/kubernetes-upgrade-tsee/helm.mdx

```diff
@@ -66,39 +66,40 @@ These steps differ based on your cluster type. If you are unsure of your cluster
        : 'curl -O -L {{downloadsurl}}/ee/charts/tigera-operator-{{chart_version_name}}.tgz'}
    </CodeBlock>
 
-1. Install the {{prodname}} custom resource definitions.
+2. Install the {{prodname}} custom resource definitions.
 
    ```bash
    kubectl apply --server-side --force-conflicts -f {{filesUrl}}/manifests/operator-crds.yaml
    kubectl apply --server-side --force-conflicts -f {{filesUrl}}/manifests/prometheus-operator-crds.yaml
    kubectl apply --server-side --force-conflicts -f {{filesUrl}}/manifests/eck-operator-crds.yaml
    ```
 
-1. Run the Helm upgrade command for `tigera-operator`:
+3. Run the Helm upgrade command for `tigera-operator` and make sure to either update `values.yaml` with your configuration or use custom `values.yaml` file:
 
-   <CodeBlock language='bash'>
-     {'{{version}}' === 'master'
-       ? (
-         `helm upgrade calico-enterprise tigera-operator-v0.0.tgz \\
-   --set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \\
-   --namespace tigera-operator`
-       )
-       : (
-         `helm upgrade calico-enterprise tigera-operator-{{chart_version_name}}.tgz \\
-   --set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \\
-   --namespace tigera-operator`
-       )
-     }
-   </CodeBlock>
+  <CodeBlock language='bash'>
+      {'{{version}}' === 'master'
+          ? `helm upgrade calico-enterprise --values=<path to values yaml> tigera-operator-v0.0.tgz \\
```

Comment:
It's clearer that you're indicating user-supplied values if you stick with this format: `<path-to-values-yaml>`.

While you're in the file, would you mind also changing to `<path-to-pull-secret>`?